tests/base.py

```diff
@@ -13,6 +13,11 @@ def assertIn(self, object, collection):
             return self.assertTrue(object in collection)
         return super(BaseTestCase, self).assertIn(object, collection)
 
+    def assertNotIn(self, object, collection):
+        if six.PY2 and sys.version_info[1] <= 6:
+            return self.assertTrue(object not in collection)
+        return super(BaseTestCase, self).assertNotIn(object, collection)
+
```

Comment:
Now that the tests use py.test, you could just use `assert x not in y` and get a really nice error message when it fails! (instead of having this wrapper)